debsources/app/templates/_macros.html

```diff
@@ -101,11 +106,53 @@
 
 {% macro render_news(news, archived_news) %}
   {% if news -%}
-    <h3><a name="news">News</a>
+  <div class="row">
+    <div class="col-md-12">
+      <h3><a name="news">News</a>
       {% if archived_news -%}
       <small><a href="{{ url_for('.news_archive') }}" name="news_archive">(News archive)</a></small>
       {% endif %}
-    </h3>
-    {{ news|safe }}
+      </h3>
+      {{ news|safe }}
+    </div>
+  </div>
   {% endif %}
 {% endmacro %}
+
+{% macro render_navbar(nav_map) %}
+  {#  render_navbar takes dictionary and produced a navbar
+      Each key refers to a link the key is directly placed into the
+      `<li></li>` while the value is to looked up by url_for()
+
+      Example Input:
+          {'Home':'home', '<span class="back">Back<span>':'back'}
+
+      Output:
+          ...
+          <li><a href="{{url_for('home')}}">Home</a></li>
+          <li><a href="{{url_for('back'}}"><span class="back">Back</span></a></li>
+          ...
+  #}
+  <div class="collapse navbar-collapse" id="ds-navbar-collapse-1">
+    <ul class="nav navbar-nav">
+    {% for key in nav_map %}
+      <li><a href="{{url_for(nav_map[key])}}">{{key}}</a></li>
+    {% endfor %}
+    </ul>
+  </div>
+{% endmacro %}
+
+{% macro remove_content_height() %}
+  {# This is an inline style that removes the content height 
```

Comment:
It's kind of a hacky fix for an style. The reason it's here is so I didn't break any features, since content height may be necessary in templates but I'm unaware.. However, if it's useless I'll remove it.